Makefile

```diff
@@ -18,9 +18,10 @@ test-fix: vendor
 update-compatibility-patch:
 	@git apply tests/php-compatibility.patch
 	@echo -e "Please open your editor and apply your changes\n"
-	@until [ "$${compatibility_resolved}" == "y" ]; do read -p "Have finished your changes (y|n)? " compatibility_resolved; done && compatibility_resolved=
+	@$$EDITOR tests/expected_report.txt
```

Comment:
Can somebody that uses Mac OS test this please?

Reply:
What would you like me to run? just `make update-compatibility-patch`?


```❯ make update-compatibility-patch
-e Please open your editor and apply your changes

/bin/sh: tests/expected_report.txt: Permission denied
make: *** [update-compatibility-patch] Error 126
```

Reply:
Looks like your version of echo doesn't support `-e`. Also, looks like you don't have a `$EDITOR` variable set, can you confirm that?

Reply:
Well yeah I confirm that. Setup should be made to deal with that though, right?

Reply:
> Setup should be made to deal with that though, right?

Can you elaborate? I don't understand what you mean.

Reply:
I mean Makefile should handle the case when $EDITOR is not defined

Reply:
Ah is see, with `EDITOR ?= vim` for instance?

Reply:
Seems a bit forceful, I will just drop that commit.